src/errorpage.cc

```diff
@@ -1350,8 +1350,8 @@ ErrorState::BuildHttpReply()
     if (request) {
         if (detail)
             request->detailError(type, detail);
-        else
-            request->detailError(type, SysErrorDetail::NewIfAny(xerrno));
+        if (const auto errNoDetail = SysErrorDetail::NewIfAny(xerrno))
```

Comment:
```suggestion
        if (const auto errnoDetail = SysErrorDetail::NewIfAny(xerrno))
```
A kind of nitpicking but clarifies a bit - since these are details of `errno`, and not the 'no details in the error'.